BCVaccineValidator/Classes/Models/DecodedQRPayload.swift

```diff
@@ -136,4 +139,21 @@ public extension DecodedQRPayload {
         return self.vc.credentialSubject.fhirBundle.entry
             .compactMap({$0.resource}).filter({$0.resourceType.lowercased() == "Immunization".lowercased()})
     }
+    
+    func isExempt() -> Bool {
+        let entries = self.vc.credentialSubject.fhirBundle.entry.filter { $0.resource.resourceType == "Condition" }
+        let conditionalEntries = entries.filter { item in
```

Comment:
Need two filter operations?

Reply:
yes just for readability, so first filter to get entries with resourceType == "Condition", and then filter the result with YukonCodingSystem